docs/migration/4_0.md

```diff
@@ -438,6 +438,8 @@ Lib: @spartacus/core
 Change: constructor parameter `checkoutDeliveryService: CheckoutDeliveryService` is removed.
 AddressFormComponent now uses the new address verification function from  `UserAddressService` called `verifyAddress` instead of the `verifyAddress` function from `CheckoutDeliveryService`.  `UserAddressService.verifyAddress` does not use the ngrx store under the hood.
 
+Change: constructor parameter `translation: TranslationService` is added
```

Comment:
I will suggest using something more common from migration doc like
`- `TranslationService` is a new, required constructor dependency.`

and than additional description